MAINTAINERS.md

```diff
@@ -0,0 +1,76 @@
+# Documentation for project maintainers (MDTP Platform UI team)
+
+## Publishing the hmrc-frontend WebJar locally
+
+In order to make the hmrc-frontend assets easy to consume in a JVM environment,
+the npm `build:webjar` task builds a JVM compatible webjar. This webjar is published to HMRC's open
+artefact repository by an internal automated deployment process and is a dependency of
+[hmrc/play-frontend-hmrc](https://www.github.com/hmrc/play-frontend-hmrc).
+
+When testing changes in conjunction with `play-frontend-hmrc` and consuming frontend microservices,
+it's possible to publish the hmrc-frontend webjar locally as follows. You will need Java
+and [Maven](https://maven.apache.org/install.html) installed.
+
+```shell script
+npm run build:package
+npm run build:webjar
+npm run publish-local:webjar
+```
+
+You can then reference the webjar in the `LibDependencies.scala` file in `play-frontend-hmrc` as follows:
+
+```sbt
+"uk.gov.hmrc.webjars" % "hmrc-frontend" % "X.Y.Z"
+```
+
+You will also need to configure your `build.sbt` resolvers to look in your local Maven repository:
+
+```sbt
+  .settings(
+    ...
+    resolvers += Resolver.mavenLocal,
+    ...
+  )
+```
+
+Further documentation on the webjar mechanism can be found:
+
+* https://www.webjars.org/documentation
+* https://www.playframework.com/documentation/2.8.x/AssetsOverview
+
+## Creating a patch release for an earlier version
+
+Occasionally, we need to release a patch to earlier major versions of the library, to fix critical issues
+or make mandatory changes for services on earlier versions that can't easily upgrade to the latest.
+An example of this was the mandated switch to use the Tudor Crown branding following the accession of King Charles III.
+
+We use the following workflow for patch releases:
+1. Create a release branch for the new version, eg.
+`git checkout -b release/v4.5.1`
+2. Create a feature branch for the required changes, eg.
+`git checkout -b PLATUI-9999-my-critical-changes`
```

Comment:
what about this, adds the tag to create the branches from, and step to push the release branch to github so it's available as a target of when you push your changes

```suggestion
1. Create a release branch for the new version, eg.
`git checkout -b release/v4.5.1 tags/4.5.0`
2. Push release branch to github, eg. `git push -u origin release/v4.5.1`
3. Create a feature branch for the required changes, eg.
`git checkout -b PLATUI-9999-my-critical-changes tags/4.5.0`
```

Reply:
updated now, kept it all explicit - would be nice to link to the build job in Jenkins, but I guess we shouldn't do this in a public repo?

Reply:
ah you're right! I didn't think of that 🙌

Reply:
yeah, I still don't understand all the rules around linking to stuff - so just assume no links 🤷‍♂️